spec/features/consumer/account/cards_spec.rb

```diff
@@ -20,28 +21,52 @@
         to_return(:status => 200, :body => JSON.generate(deleted: true, id: "cus_AZNMJ"))
     end
 
-    it "lists saved cards, shows interface for adding new cards" do
+    it "passes the smoke test" do
       visit "/account"
 
       click_link I18n.t('spree.users.show.tabs.cards')
 
       expect(page).to have_content I18n.t(:saved_cards)
 
+      # Lists saved cards
       within(".card#card#{card.id}") do
         expect(page).to have_content card.cc_type.capitalize
         expect(page).to have_content card.last_digits
+        expect(find_field('default_card')).to be_checked
       end
 
+      within(".card#card#{card2.id}") do
+        expect(page).to have_content card2.cc_type.capitalize
+        expect(page).to have_content card2.last_digits
+        expect(find_field('default_card')).to_not be_checked
+      end
+
+      # Allows switching of default card
```

Comment:
I prefer this approach with integration specs, particularly when the full UI is being loaded and interacted with using Capybara. It's just a more pragmatic approach to testing the things that need to be tested. In an ideal world with infinite resources, yes, we would load up the entire app to test every small thing in a feature spec, but if we did that our test suite would take 8 hours to run. I prefer to use a "smoke test" to check that a bunch of stuff is generally working on a page, and then have a couple of dedicated specs for really vital/important features. The use of the comments is designed to still allow other developers to understand what is being tested within each section, so that they can understand what is broken. IMHO this is not really different to the description of a test example, except that you don't get the message in your test output, you have to go to the file find it. I think this is an acceptable tradeoff for the reduced run time of the test suite. 

Obviously it's a totally different with unit specs, and even integration specs which don't need to load the entire app in order to run (controller specs/request specs). In those cases absolutely we should have dedicated examples which test one thing in a structured way.